support/client/lib/mil-sym/cws.js

```diff
@@ -5288,10 +5338,78 @@ define( function(){
                   "symbolID": "G*GPOLKA--****X",
                   "tag": "DIRECTION OF ATTACK AVIATION",
                   "symbolType": "SimpleArrow"
+            },
+            "TACGRP.C2GM.SPL.ARA.AOO": {
+                  "symbolID": "G*GPSAO---****X",
+                  "tag": "AREA OF OPERATIONS",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.SPL.ARA.AHD": {
+                  "symbolID": "G*GPSAA---****X",
+                  "tag": "AIRHEAD",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.SPL.ARA.ENCMT": {
+                  "symbolID": "G*GPSAE---****X",
+                  "tag": "ENCIRCLEMENT",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "hostile" ]
```

Comment:
This is listed in the modifiers dialog, but the input is disabled (same for all the graphics that have a `hostile` modifier).

Reply:
Yup.

Reply:
I'm guessing that's not the intended behavior.  :wink:  What are we hoping that will do?